ElrondNetwork/elrond-go
# Fix how gasUsed is calculated on move balance transaction

Fix how gas used is calculated for move balance transactions.
Now gasUsed for a move balance transaction  is calculated with formula `minGasLimit + len(data)*gasPerDataByte`

Testing procedure:
 1.  run a testnet with this branch and make sure that external.toml has ElasticSearchConnector enabled
 2. in epoch 0:generate an invalid transaction with `gasLimit= 500 000` (to generate an invalid transaction value from transaction  
       should be greater than sender account balance.
 2. check in elasticsearch database that gasUsed of generated transaction is equals with `50 000`

 3. wait 2 epochs to pass (when you move to next step epoch number  should be greater of equal with 2)
 4. generate again an invalid transaction with gasLimit= `500 000`
 5. check in elasticsearch database that gasUsed of generated transaction is equals with `500 000
`
